packages/server/test/unit/plugins/child/run_require_async_child_spec.js

```diff
@@ -17,28 +15,6 @@ describe('lib/plugins/child/run_require_async_child', () => {
     mockery.deregisterMock('@cypress/webpack-batteries-included-preprocessor')
   })
 
-  describe('typescript registration', () => {
-    beforeEach(() => {
-      sinon.stub(tsNodeUtil, 'register')
-      sinon.stub(resolve, 'typescript').returns('/path/to/typescript.js')
-    })
-
-    it('registers ts-node only once when typescript module found', function () {
-      runRequireAsyncChild(this.ipc, 'cypress.config.js', 'proj-root')
-      runRequireAsyncChild(this.ipc, 'cypress.config.js', 'proj-root')
-
-      expect(tsNodeUtil.register).to.be.calledWith(
-        'proj-root',
-        'cypress.config.js',
-      )
-
-      expect(tsNodeUtil.register).to.be.calledOnce
-    })
-
-    // FIXME: need to validate that TS is checked once when ts is not found as well
-    it.skip('checks for typescript only once if typescript module was not found')
-  })
-
```

Comment:
We don't register ts-node in the child process anymore, instead, we run the child process with `--require server/lib...../register_ts_node`. This also means we don't need the check to avoid the double registration anymore.